mergify_engine/actions/review.py

```diff
@@ -48,7 +48,7 @@ async def run(
     ) -> check_api.Result:
         if self.config["message"] is None:
             return check_api.Result(
-                check_api.Conclusion.SUCCESS, "Message is not set", ""
+                check_api.Conclusion.FAILURE, "Message is not set", ""
```

Comment:
```suggestion
                check_api.Conclusion.FAILURE, "Message is not set", "The `review` action does not have any message configured."
```

or I'd do this:
```python
message = self.config["message"] or "👀"
```
if we want to make it work by default
maybe use 👍  / 👎  / 💬 by default depending on the `type`?

Reply:
I just see it's already handled some lines later.